editor/plugins/sprite_2d_editor_plugin.cpp

```diff
@@ -597,6 +642,26 @@ Sprite2DEditor::Sprite2DEditor() {
 
 	options->get_popup()->connect(SceneStringName(id_pressed), callable_mp(this, &Sprite2DEditor::_menu_option));
 
+	// Resize region rect definition
+	resize_region_rect = memnew(Button);
+
+	top_hb->add_child(resize_region_rect);
```

Comment:
Add child _after_ doing setup. This avoids unnecessary notifications and callbacks when node is being setup while in tree.